README.md

```diff
@@ -126,6 +126,20 @@ Run the Maven target 'jetty:stop'. If that does not help, use your opering syste
 
     kill -9 `lsof -i :8080 -t`
 
+## 9. Third-party software
+For some modules in Molgenis, third-party software is in use. It is important to know that some of these licenses are different than the Molgenis license.
```

Comment:
Complete list of licence models of dependencies known to maven is generated in molgenis-reports and sits here: http://www.molgenis.org/jenkins/job/molgenis-reports/site/dependencies.html#Licenses

That does not include javascript libraries, but is otherwise pretty thorough.